airflow-core/src/airflow/dag_processing/bundles/manager.py

```diff
@@ -152,8 +166,6 @@ def parse_config(self) -> None:
 
         If a bundle class for a given name has already been imported, it will not be imported again.
 
-        todo (AIP-66): proper validation of the bundle configuration so we have better error messages
```

Comment:
Hey @jedcunningham. When working on this one I took the liberty to use Pydantic to specify the dag bundle config schema. Let me know if this is what you meant by "proper validation of the bundle configuration so we have better error messages". Happy to revisit if you had something else in mind.

Reply:
LGTM, thanks.